### DISCUSSION TITLE
Could this extension help to install dbg when you need to find a core dump reason ?

### DISCUSSION THREAD

**Discussion by vasilvestre** [mlocati/docker-php-extension-installer#942] [General]
I'm trying to install dbg but most documentation are about ubuntu, ppa/ondrej and nothing about alpine. I'm having a hard time to fix these.
Maybe this installed can help but I'm not sure

**Comment by mlocati** [comment]
This project is about the install-php-extensions script, which installs PHP extensions: installing system utilities is out of scope...

Have you tried simply running the following?

```
apk add gdb
```

**Reply by vasilvestre** [reply]
Yeah but maybe I miss symbols and must reconfigure PHP with debug enabled. It's a bit tricky 😓 
Thank you :)
